funcx_endpoint/funcx_endpoint/tests/test_manager.py

```diff
@@ -0,0 +1,54 @@
+from funcx_endpoint.endpoint.manager import EndpointManager
+import os
+import logging
+import sys
+import shutil
+
+logger = logging.getLogger('mock_funcx')
+
+def test_configure():
+    manager = EndpointManager(logger)
+    manager.FUNCX_DIR = f'{os.getcwd()}'
+    config_dir = os.path.join(manager.FUNCX_DIR, "mock_endpoint")
+
+    try:
+        manager.configure_endpoint("mock_endpoint", None)
```

Comment:
Also need to test what happens when `configure_endpoint` called with an existing config